task/helm-upgrade-from-repo/0.1/helm-upgrade-from-repo.yaml

```diff
@@ -46,4 +46,4 @@ spec:
         echo adding updating repo...
         helm repo update
         echo installing helm chart...
-        helm upgrade --wait --install --namespace "$(inputs.params.release_namespace)" $(inputs.params.release_name) $(inputs.params.chart_name) --debug --set "$(inputs.params.overwrite_values)"
\ No newline at end of file
+        helm upgrade --wait --install --namespace "$(inputs.params.release_namespace)" $(inputs.params.release_name) --debug --set "$(inputs.params.overwrite_values)"
```

Comment:
Please remove the inputs from params, while you are at it

Reply:
once the CI is fix then probably I can just bump up the new version and mark this as deprecated :slightly_smiling_face: